cmd/query/app/util.go

```diff
@@ -15,9 +15,20 @@
 package app
 
 import (
+	"time"
+
 	"github.com/jaegertracing/jaeger/storage/spanstore"
 )
 
+// clock is a wrapper around the time package to allow mocking of the current time.
```

Comment:
You can avoid this type by using the function signature directly `func() time.Time`, and then pass `time.Now` as main impl